calico/felix/fiptables.py

```diff
@@ -383,6 +406,7 @@ def _calculate_ipt_delete_input(self, chains):
         if found_delete:
             return input_lines
         else:
+            #PLW: Nothing seems to catch this exception (unlike from modify)
```

Comment:
I think we'll never get here due to the way batches are calculate.  That said, no harm in adding the except: block in the right place.